### PR TITLE
zh-cn: Format /web/css using Prettier (part 4)

### PR DESCRIPTION
**Pull request by queengooborg**
This PR formats the `/web/css` folder of the Simplified Chinese locale using Prettier.  This is the fourth part.

### CODE REVIEW

**Review by yin1999 — COMMENTED** [review]
Just some fixes before we can land this ;)

File: files/zh-cn/web/css/word-spacing/index.md [hunk 1]
```diff
@@ -12,11 +12,8 @@ slug: Web/CSS/word-spacing
 ## 语法
 
 ```html
-word-spacing: normal;    /* Keyword value */
-word-spacing: 3px;       /* <length> values */
-word-spacing: 0.3em;
-
-word-spacing: inherit;
+word-spacing: normal; /* Keyword value */ word-spacing: 3px; /*
+<length> values */ word-spacing: 0.3em; word-spacing: inherit;</length>
```
[the remark this answers is not in the document]
Comment (queengooborg) [reply files/zh-cn/web/css/word-spacing/index.md:16]: Oops, how'd I miss this one?

**Review by yin1999 — APPROVED** [review]
Thank you @queengooborg 😊

File: files/zh-cn/web/css/word-spacing/index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (yin1999) [reply files/zh-cn/web/css/word-spacing/index.md]: Doesn't matter. This is really a hard job 💪

**Pull request merged**